src/main/java/aeneas/controllers/SaveLevelController.java

```diff
@@ -1,24 +1,36 @@
 package aeneas.controllers;
 
 import aeneas.models.Level;
-import aeneas.views.BuildLevelView;
+
+import java.io.File;
+import java.io.IOException;
 
 import javafx.event.EventHandler;
 import javafx.scene.input.MouseEvent;
+import javafx.stage.FileChooser;
+import javafx.stage.Stage;
 
 // Not sure what sort of event this ends up taking.
 public class SaveLevelController implements EventHandler<MouseEvent> {
 
   Level levelModel;
-  BuildLevelView view;
+  Stage mainStage;
 
-  public SaveLevelController(BuildLevelView view, Level levelModel){
+  public SaveLevelController(Stage mainStage, Level levelModel){
     this.levelModel = levelModel;
-    this.view = view;
+    this.mainStage = mainStage;
   }
 
   @Override
   public void handle(MouseEvent event) {
+    FileChooser fileChooser = new FileChooser();
+    fileChooser.setTitle("Save Level");
+    File saveFile = fileChooser.showSaveDialog(mainStage);
+    try {
+      levelModel.save(saveFile);
+    } catch (IOException i) {
+      System.out.println("Error occurred in opening file.");
+    }
```

Comment:
I don't like passing the state around. Can we make methods on MainView to open/save files?